src/eventSchemas/commands/searchSchemas.ts

```diff
@@ -86,8 +87,11 @@ export async function createSearchSchemasWebView(params: { node: RegistryItemNod
             ext.context.subscriptions
         )
     } catch (err) {
+        webviewResult = 'Failed'
         const error = err as Error
         logger.error('Error searching schemas', error)
+    } finally {
+        recordSchemasSearch({ result: webviewResult })
```

Comment:
Correct, adding it to the catch block of `getSearchListForSingleRegistry` should cover the case of failed search query. Side note, I realized in `fetchSchemaContent` case,  describeSchema api call should also be surrounded by try/catch as well. That way we can record the telemetry for failed describe calls, as well as notifying the user that fetching content failed rather than displaying  'Loading' as a schema content. Will do it as a follow pr.

Reply:
The telemetry was being ran in the caller of this, this PR was not intended to change where we do existing telemetry.

Reply:
@bryceitoc9 nice find. @ejafarli can you please take a task to add any missing metrics calls as needed?

Reply:
Definitely, added it to the follow up tasks. Will create a SIM ticket as well.